app/src/main/java/com/keylesspalace/tusky/fragment/NotificationsFragment.java

```diff
@@ -330,10 +331,10 @@ private void updateFilterVisibility() {
 
     private void confirmClearNotifications() {
         new AlertDialog.Builder(requireContext())
-                .setMessage(R.string.notification_clear_text)
-                .setPositiveButton(android.R.string.ok, (DialogInterface dia, int which) -> clearNotifications())
-                .setNegativeButton(android.R.string.cancel, null)
-                .show();
+            .setMessage(R.string.notification_clear_text)
+            .setPositiveButton(android.R.string.ok, (DialogInterface dia, int which) -> clearNotifications())
+            .setNegativeButton(android.R.string.cancel, null)
+            .show();
```

Comment:
Can you maybe remove these indendation changes?

Reply:
for sure, sorry!